core/nginx/nginx.conf.template

```diff
@@ -74,6 +74,7 @@ http {
       proxy_set_header Authorization "Basic %(auth)s";
     }
     location = /%(urlhash)s/beaker/ {
+      autoindex off;
```

Comment:
Word, sometimes it is nice to split these changes up into separate commits. Take for instance this case.

You open a PR with one commit that does 3 things. 

1 of the things causes a bug or regression.
Since it is all in one commit we can not git revert <SHA> without losing the other features.

If it is a single commit per "feature or change" it makes it easy to back out mistakes.